sdv/tabular/ctgan.py

```diff
@@ -1,7 +1,7 @@
 """Wrapper around CTGAN model."""
 
 import numpy as np
-from ctgan import CTGANSynthesizer, TVAESynthesizer
+from ctgan import CTGAN, TVAE
```

Comment:
I think that it would be better to just import `ctgan` and use `ctgan.CTGAN` and `ctgan.TVAE` in the code, just to avoid messing up the names.
eg:
`from sdv.tabular.ctgan import CTGAN` it becomes a bit unclear whether it will import the `CTGAN` class from `ctgan` or the `CTGAN` class declared in here.